sites/telecablesat.fr/telecablesat.fr_fr.channels.xml

```diff
@@ -0,0 +1,288 @@
+    <channel lang="fr" xmltv_id="" site_id="308">France 3 Corse Via Stella</channel>
+    <channel lang="fr" xmltv_id="" site_id="1932">France 3 Franche-Comté</channel>
+    <channel lang="fr" xmltv_id="" site_id="1934">France 3 Languedoc-Roussillon</channel>
+    <channel lang="fr" xmltv_id="" site_id="1935">France 3 Limousin</channel>
+    <channel lang="fr" xmltv_id="" site_id="1936">France 3 Lorraine</channel>
+    <channel lang="fr" xmltv_id="" site_id="1937">France 3 Midi-Pyrénées</channel>
+    <channel lang="fr" xmltv_id="" site_id="1938">France 3 Nord Pas-de-Calais</channel>
+    <channel lang="fr" xmltv_id="" site_id="1925">France 3 Normandie - Caen</channel>
+    <channel lang="fr" xmltv_id="" site_id="1933">France 3 Normandie - Rouen</channel>
+    <channel lang="fr" xmltv_id="" site_id="1939">France 3 Paris Ile-de-France</channel>
+    <channel lang="fr" xmltv_id="" site_id="1940">France 3 Pays de la Loire</channel>
+    <channel lang="fr" xmltv_id="" site_id="1941">France 3 Picardie</channel>
+    <channel lang="fr" xmltv_id="" site_id="1942">France 3 Poitou-Charentes</channel>
+    <channel lang="fr" xmltv_id="" site_id="1943">France 3 Provence-Alpes</channel>
+    <channel lang="fr" xmltv_id="" site_id="1944">France 3 Rhône-Alpes</channel>
+    <channel lang="fr" xmltv_id="" site_id="78">France 4</channel>
+    <channel lang="fr" xmltv_id="" site_id="47">France 5</channel>
+    <channel lang="fr" xmltv_id="" site_id="2111">Franceinfo</channel>
+    <channel lang="fr" xmltv_id="" site_id="87">Game One</channel>
+    <channel lang="fr" xmltv_id="" site_id="1166">Golf Channel</channel>
+    <channel lang="fr" xmltv_id="" site_id="1295">Golf+</channel>
+    <channel lang="fr" xmltv_id="" site_id="329">Guadeloupe la 1ère</channel>
+    <channel lang="fr" xmltv_id="" site_id="482">Gulli</channel>
+    <channel lang="fr" xmltv_id="" site_id="260">Guyane la 1ère</channel>
+    <channel lang="fr" xmltv_id="" site_id="88">Histoire TV</channel>
+    <channel lang="fr" xmltv_id="" site_id="416">Hustler TV</channel>
+    <channel lang="fr" xmltv_id="" site_id="781">I24news</channel>
+    <channel lang="fr" xmltv_id="" site_id="94">Infosport+</channel>
+    <channel lang="fr" xmltv_id="" site_id="1585">J-One</channel>
+    <channel lang="fr" xmltv_id="" site_id="1280">Ketnet</channel>
+    <channel lang="fr" xmltv_id="" site_id="110">KTO</channel>
+    <channel lang="fr" xmltv_id="" site_id="1401">L&apos;Equipe</channel>
+    <channel lang="fr" xmltv_id="" site_id="234">La Chaîne parlementaire</channel>
+    <channel lang="fr" xmltv_id="" site_id="892">La Trois</channel>
+    <channel lang="fr" xmltv_id="" site_id="164">La Une</channel>
+    <channel lang="fr" xmltv_id="" site_id="112">LCI - La Chaîne Info</channel>
+    <channel lang="fr" xmltv_id="" site_id="535">LM TV Sarthe</channel>
+    <channel lang="fr" xmltv_id="" site_id="118">M6</channel>
+    <channel lang="fr" xmltv_id="" site_id="683">Man-X</channel>

```

Comment:
```suggestion
    <channel lang="fr,en" xmltv_id="" site_id="683">Man-X</channel>
```
European-scaled gay adult channel. Cannot confirm if French only, as the programmation is written in English. I do not have this channel myself, ISP providing me only with channels like Brazzers, Penthouse or Dorcel.

Reply:
Strangely, my program is in French:

<img width="853" alt="image" src="https://user-images.githubusercontent.com/7253922/157729844-fb2139d7-f7f5-4753-ab07-7cf52d125d83.png">

And in this file you can not specify multiple languages at the same time.

Reply:
Had a less detailed programmation sorry, firewall blocks this website here.

![image](https://user-images.githubusercontent.com/30985701/157731404-185a8413-cab3-4564-ba44-424514b8804e.png)